src/automations/types.ts

```diff
@@ -0,0 +1,35 @@
+export interface IAutomation {
+  cancelation_token?: string;
+  steps: any[];
```

Comment:
could we be more explicit here?

Reply:
added types for all types of step we support. No pun intended :D